src/main/java/edu/kit/hci/soli/domain/SystemConfiguration.java

```diff
@@ -0,0 +1,45 @@
+package edu.kit.hci.soli.domain;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.ToString;
+
+/**
+ * Entity class representing the system configuration.
+ * This table is designed to contain only a single row.
+ */
+@Entity
+@Table(name = "soli_system_configuration")
+@ToString
+public class SystemConfiguration {
```

Comment:
I'm not sure if this is correct approach to storing application settings... But what other alternatives do we have? I guess this is fine

Reply:
I agree that this feels dirty, but according to StackOverflow, doing this should be just fine if we add a database constraint to maintain the singleton nature (which I have done)